frontend/packages/console-dynamic-plugin-sdk/src/app/AppInitSDK.tsx

```diff
@@ -1,31 +1,49 @@
 import * as React from 'react';
 import { Provider } from 'react-redux';
+import { Store } from 'redux';
+import { ResourceFetch, setUtilsConfig } from './configSetup';
 import { useReduxStore } from './useReduxStore';
 
 type AppInitSDKProps = {
   children: React.ReactNode;
+  configurations: {
+    apiDiscovery: (store: Store<any>) => void;
+    appFetch: ResourceFetch;
```

Comment:
Maybe for sanity, we go with an indirect type:

```suggestion
    appFetch: UtilsConfig['appFetch'];
```